tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_then_reset.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.clock.app import Clock
+import time
+
+class TestClockRunStopWatch(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        self.clock = Clock(self.marionette)
+        self.clock.launch()
+
+    def test_click_run_stopwatch_then_reset(self):
+
+        stopwatch_view = self.clock.switch_view("stopwatch")
+
+        time.sleep(0.2)
+        self.assertEqual(stopwatch_view.current_time, '00:00.00')
+
+        stopwatch_view.tap_start()
+
+        self.assertNotEqual(stopwatch_view.current_time, '00:00.00')
+
+        first_time = stopwatch_view.current_time
```

Comment:
I think we want this after line 30, stopwatch_view.tap_pause(), we want to check if the pause button works correctly, so that after this time.sleep of 2 seconds, we still have the same current_time on the stopwatch.
We already checked that the stopwatch was working after the tap_start, so we don't need another check for that.